### DISCUSSION TITLE
Svg icon

### DISCUSSION THREAD

**Discussion by Niccolum** [sqlalchemy/sqlalchemy#7672] [Usage Questions] [answered]
SQLAlchemy is awesome project with awesome history. But it still doesn't have official svg icon. Can someone create official logo in svg format? To use it, fore example, [here](https://github.com/simple-icons/simple-icons/issues/5346) ?)

**Comment by zzzeek** [comment]
ive always been antsy about that.   do other projects have svgs of their logos freely available?   can you show me some and how they are presented?

**Reply by Niccolum** [reply]
.NET
https://github.com/dotnet/brand/tree/main/logo

1password
https://1password.com/press/

Django
https://www.djangoproject.com/community/logos/

FastAPI
https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/img/icon-white.svg

More examples [here](https://github.com/simple-icons/simple-icons/blob/develop/_data/simple-icons.json)

**Reply by zzzeek** [reply, marked as answer]
i dont have time to present on the site so here it is 
![sqlalchemy_logo](https://user-images.githubusercontent.com/128223/153606192-c117618d-cabf-49c4-9204-d7f1afe24289.svg)
